pep_sphinx_extensions/pep_zero_generator/pep_index_generator.py

```diff
@@ -19,13 +19,20 @@
 
 import json
 import os
+import sys
 from pathlib import Path
 from typing import TYPE_CHECKING
 
 from pep_sphinx_extensions.pep_zero_generator import parser
 from pep_sphinx_extensions.pep_zero_generator import subindices
 from pep_sphinx_extensions.pep_zero_generator import writer
 from pep_sphinx_extensions.pep_zero_generator.constants import SUBINDICES_BY_TOPIC
+if sys.version_info >= (3, 11):
+    from release_engineering.generate_release_cycle import create_release_cycle
+else:
+    # this function uses tomllib, which requires Python 3.11+
```

Comment:
The comment would make more sense before line 31 or before line 30. _This_ function here on lines 34-35 does not use tomllib!

But also, you could be a version-dependent dependency like `tomli >= 1.1.0 ; python_version < "3.11"` and a try/except import. But it’s up to you, I’m not well versed in this repo to know whether RMs would use older pythons, and then whether a silent no-op is better than working or having a clear error 🙂